lms/djangoapps/edxnotes/plugins.py

```diff
@@ -27,4 +28,8 @@ def is_enabled(cls, course, user=None):
         """
         if not super(EdxNotesTab, cls).is_enabled(course, user=user):
             return False
+
+        if settings.FEATURES.get("ENABLE_EDXNOTES") is False or is_harvard_notes_enabled(course):
+            return False
```

Comment:
If I look at `helpers.py:is_feature_enabled()`, notes is only enabled if `is_harvard_notes_enabled()` returns `False`. Why are we doing the opposite here?